### PR TITLE
Update 05-trust-contracts.md

### PR DESCRIPTION
**Pull request by cjweisman**
Fixed header typo. Completed the section

### CODE REVIEW

**Comment by kchulj** [comment]
Thank you for the prompt update. Brian and I will have a better look at it tomorrow.

Fixes #9

File: docs/source/05-trust-contracts.md [hunk 1]
```diff
@@ -14,4 +14,4 @@ Trust Contracts enable programming of decidable contracts, either by using a fle
 
 To overcome performance limitation inherent in blockchain, Trust Contracts utilize [WebAssembly](https://developer.mozilla.org/en-US/docs/WebAssembly). WebAssembly provides a way to run code, written in multiple languages, on the web at near native speed, with client apps running on the web that previously couldn’t have done so.
 
-The ultimate goal of this architecture is to be able users to build a de
+The ultimate goal of this architecture is to enble users to build a dedecidable contract, which ensures safe and accurate execution, while maximizing its scalability.
```
Comment (hewison-chris) [inline docs/source/05-trust-contracts.md:17]:
```suggestion
The ultimate goal of this architecture is to enable users to build a decidable contract, which ensures safe and accurate execution, while maximizing its scalability.
```

**Review by hewison-chris — CHANGES_REQUESTED** [review]
A couple of typos.

**Comment by kchulj** [comment]
Ping @bpalaggi for review

**Review by kchulj — APPROVED** [review]
LGTM
Thank you @hewison-chris

**Comment by bpalaggi** [comment]
LGTM thanks @kchulj @cjweisman

**Review by bpalaggi — APPROVED** [review]

File: docs/source/05-trust-contracts.md (hunk 1 above)
Comment (hewison-chris) [reply docs/source/05-trust-contracts.md:17]: Looks like this suggestion was not applied to fix the typos.

**Pull request merged**